### ISSUE TITLE
Index.js doesn't creating

### ISSUE THREAD

**Issue by krakotay** [napi-rs/napi-rs#1768]
Hi all, I got an Error
`PS C:\Users\krakotay\Documents\ts\for_profi\rust + TS\excel_to_csv> yarn build
    Finished release [optimized] target(s) in 0.02s
(node:944) [DEP0040] DeprecationWarning: The `punycode` module is deprecated. Please use a userland alternative instead.
(Use `node --trace-deprecation ...` to show where the warning was created)
Syntax Error: Unexpected end of JSON input
    at JSON.parse (<anonymous>)
    at C:\Users\krakotay\Documents\ts\for_profi\rust + TS\excel_to_csv\node_modules\@napi-rs\cli\scripts\index.js:11662:50
    at Array.map (<anonymous>)
    at C:\Users\krakotay\Documents\ts\for_profi\rust + TS\excel_to_csv\node_modules\@napi-rs\cli\scripts\index.js:11662:31
    at Generator.next (<anonymous>)
    at fulfilled (C:\Users\krakotay\Documents\ts\for_profi\rust + TS\excel_to_csv\node_modules\@napi-rs\cli\scripts\index.js:3549:56)`

It start every time that I build any project, even the "Hello world"
And even tomorrow all worked fine. How I can resolve it?

**Comment by Brooooooklyn** [comment]
@krakotay I'll fix it in the next `napi-build` release, you can run the `cargo clean` and rebuild project as a workaround

**Comment by laozhoubuluo** [comment]
Now it seems that this problem still occurs on the latest 2.1.2 version.

**Comment by wiredprairie** [comment]
Maybe this will help someone else ...

I was having this problem in a completely new Rust project and here's what I did (the project was not using `yarn`):

1. Confirmed using latest version of this crate: `cargo add napi@2.16.6`
2. I'd locally installed `napi-cli`, so in the script to start the build within `package.json`:

```json
        "build": "npm exec napi build -- --platform --release",
        "build:debug": "npm exec napi build -- --platform",
```
It had been `yarn`, but `yarn` passes arguments differently than `npm exec`, so I had to add the extra `--` to pass `--platform`. 

With those two changes, the `index.js` file was created and worked when tested.
